docs/features/embed-pages-to-teams.md

```diff
@@ -203,4 +205,4 @@ See more information on these options from the following Microsoft Teams documen
 
 ## See also
 
-- [Building Microsoft Teams tab using SharePoint Framework](https://docs.microsoft.com/sharepoint/dev/spfx/web-parts/get-started/using-web-part-as-ms-teams-tab)
\ No newline at end of file
+- [Building Microsoft Teams tab using SharePoint Framework](https://docs.microsoft.com/sharepoint/dev/spfx/web-parts/get-started/using-web-part-as-ms-teams-tab)
```

Comment:
Please use relative links to pages in this repo. This will ensure the build agent will verify there are no broken links. It doesn't do that for relative links.

Reply:
Updated the relative link.